db/migrations/20230227080635_creator.sql

```diff
@@ -0,0 +1,12 @@
+-- migrate:up
+CREATE TABLE creator(
+  id INT NOT NULL AUTO_INCREMENT,
+  name VARCHAR(200) NOT NULL,
+  image_url VARCHAR(500) NULL,
+  description VARCHAR(5000) NOT NULL,
```

Comment:
varchar가 3000이 넘으면 text로 해주세요.
```suggestion
  description TEXT NOT NULL,
```